specs/v1.yaml

```diff
@@ -452,6 +452,190 @@ paths:
           $ref: "#/components/responses/NotFoundError"
         "500":
           $ref: "#/components/responses/InternalServerError"
+  /v1/recurring_grants:
+    get:
+      summary: List recurring Grants
+      description: |-
+        List all recurring grants for the provided API Key. This API allows for paginating over many results.
+      operationId: list-recurring-grants
+      tags:
+        - Grants, Recurring
+      security:
+        - auth0:
+            - "read:recurring_grants"
+      parameters:
+        - name: x-chariot-api-key
+          in: header
+          description: the apiKey of the Connect object
+          schema:
+            type: string
+          required: true
+          example: "live_xJd0lUrvpDkzeGBWZbuI2wbvEdM"
+        - name: pageLimit
+          in: query
+          description: the number of results to return; defaults to 10, max is 100
+          schema:
+            type: integer
+            default: 10
+        - name: pageToken
+          in: query
+          description: |-
+            A token to use to retrieve the next page of results. This is useful for paginating over many pages of results. If set, all other arguments are expected to be kept the same as previous calls and the value of this field should be from the nextPageToken in the previous response.
+          schema:
+            type: string
+      responses:
+        "200":
+          $ref: "#/components/responses/ListRecurringGrantsResponse"
+        "400":
+          $ref: "#/components/responses/BadRequestError"
+        "401":
+          $ref: "#/components/responses/AuthenticationError"
+        "403":
+          $ref: "#/components/responses/ForbiddenError"
+        "500":
+          $ref: "#/components/responses/InternalServerError"
+    post:
+      summary: Create Recurring Grant
+      description: |-
+        Create a recurring grant from a workflow session. This is useful to capture a recurring grant intent from an authorized connect workflow session and submit the recurring grant request.
+        The recurring grant must be captured within 5 minutes of authorization otherwise the request will return status 410 Gone.
+        A recurring grant can only be captured once from any given workflow session so any duplicate requests will return status 409 Conflict.
+        The recurring grant amount must be in whole dollar increments (rounded to the nearest hundred) as currently DAFs only accept whole dollar grants.
+        The recurring grant amount must be greater than or equal to the minimum recurring grant amount for the DAF otherwise the request will return status 400 Bad Request.
```

Comment:
it feels repetitive to keep saying "the recurring grant amount". also, i don't think there's such thing as a "minimum recurring grant amount for the DAF" - DAFs just have a minimum grant amount. so the amount that is specified for the recurring grant needs to be equal to or greater than the DAF's minimum grant amount.